### DISCUSSION TITLE
Login interconnected and email requirements

### DISCUSSION THREAD

**Discussion by brunocalmon** [mastodon/mastodon#19204] [General]
Currently we need to have several accounts without any connection with each other if we want to be registered in more than one server.

I was thinking, why can't we have a link between all accounts we want to be linked?

The proposal is:
Every account can be linked with one or more unique existents accounts that you are the owner in other servers - basically a login to an external server through the current one -, or pre-linked during the registration.

When someone link the account A with B,C,D... that means that your account now are referenced to others, every linked account in every server has an updated list of their siblings, so they can sync themselves when some of them receive any updates.

It also simulate the total of followers/followed among every linked accounts, and also, any toots from any of your linked account will be delivered and visible as it was one single account to you and your followers, if they follow at least one of your linked accounts, and you can also see their toots.

That would be a nice approach to increase the decentralization and avoid censorship.

If one of your accounts server go off, you still have all the others synced and will only lose the info from that specific server... you can normally log into any of the server you have your account normally as you already do but they will work as one.

Plus: about the email,

As I read here on the community, the reason fornthe email are to avoid spams and fake account, and I totally understand that. Nevertheless, would be nice on this case to be able to remove the email after the registration get approved, and let the user login using their username only?

I would love an option "don't keep email persisted after registration complete" during the registration.

That makes sense of the idea is only to avoid bots and spams.

**Comment by Sophist-UK** [comment]
I am pretty technical but even I am confused by how the logins are supposed to work.

I am cool with the idea of people setting up their own servers and it being a distributed social platform.

I am also cool with the idea that a user registers with one server (his home server) that fits with the individuals interests.

But when I click a link to another server, I am not logged in there, so if I want to comment on a post made by a user on another server apparently I can't with my existing login.

Am I supposed to create another login and use the profile alias functionality to make them appear as if they are one user, Or is this some sort of bug and the logins are supposed to be federated and be single sign-in?
